PickMeUp/PickMeUp/viewControllers/DM Features/PMConversationViewController.m

```diff
@@ -70,34 +91,42 @@ -(void)runGetQuery {
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
     PMDMCell *cell = [tableView dequeueReusableCellWithIdentifier:kCellIdentifier forIndexPath:indexPath];
-    cell.content.text = self.arrayOfDMs[indexPath.row].content;
+    [cell prepareForReuse];
```

Comment:
Oops. You don't want to call prepareForReuse method manually. It is called by the tableView when the cell has been used (shown) and is going back to the recycling queue. Normally you override this method and reset cell components to some default state, which seems like what you want. So try to remove this call, and set a breakpoint in prepareForReuse method, see if it's called. Most likely your code will be still doing what you want.

Let's move this cell presentation customization code into cell class and have -(void) configureWithUser:(PFUser *)user method there